app/pb_kits/playbook/pb_popover/docs/_popover_list.html.erb

```diff
@@ -0,0 +1,12 @@
+<div style="height: 750px; text-align: center;">
+<%= pb_rails("button", props: { text: "Button Secondary", variant: "secondary", id: 'list' }) %>
+<%= pb_rails("popover", props: {buttonId: "list", tooltipId: "list-tooltip", position: 'bottom', offset: false }) do %>
```

Comment:
ButtonID seems to limit this to only be buttons

Reply:
What is offset for?

Reply:
Not sure what we want to name it. ElementId? It's the element that is going to be clicked... definitely don't want the user to be pinned in by the terminology. Any other ideas? Basically the prop we need here is the ID of the "clickable trigger element". Trigger? TriggerId?

Reply:
> What is offset for?

Offset provides the spacing between the clickable trigger element and the popover element. Seen here: https://popper.js.org/docs/v2/modifiers/offset/

Reply:
> Not sure what we want to name it. ElementId? It's the element that is going to be clicked... definitely don't want the user to be pinned in by the terminology. Any other ideas? Basically the prop we need here is the ID of the "clickable trigger element". Trigger? TriggerId?

TriggerElementId